src/managers/auth-manager.js

```diff
@@ -188,7 +188,7 @@ AuthManager.prototype = $.extend(AuthManager.prototype, {
             };
 
             if (!isTeamMember) {
-                me.getUserGroups({ userId: userInfo.user_id, token: token }, userGroupOpts)
+                me.getUserGroups({ userId: userInfo.user_id, token: token, server: adapterOptions.server }, userGroupOpts)
```

Comment:
is the issue that `getUserGroups` was always calling the wrong server? If so, then the fix for that should actually go in the `getUserGroups` function; i.e., merge in `this.options` in there, something like

`        var adapterOptions = this.sessionManager.getMergedOptions({ success: $.noop }, this.options, options);`